libvast/vast/address_synopsis.hpp

```diff
@@ -90,6 +90,11 @@ class buffered_address_synopsis : public synopsis {
     ips_.insert(*addr_view);
   }
 
+  size_t size_bytes() const override {
+    return sizeof(buffered_address_synopsis)
+      + ips_.size()*sizeof(typename decltype(ips_)::node_type);
```

Comment:
```suggestion
      + ips_.size() * sizeof(typename decltype(ips_)::node_type);
```

Two minor notes:
- The formatting (spaces around binary operators)
- Should this be `ips_.size_bytes()` instead`

Reply:
Here `ips_` is just a plain `std::unordered_map`, so it doesn't have a `size_bytes()` function and we have to estimate size as number of nodes times the byte size per node.

Reply:
Ah fair enough, should've checked that.